Makefile

```diff
@@ -177,10 +178,11 @@ deploy-test:
 	&& export DUMMY_IGNITION=${DUMMY_IGNITION} && ISO_CREATION=minikube-local-registry/assisted-iso-create:minikube-test $(MAKE) update-minikube deploy-all
 
 deploy-onprem:
+	$(shell printf "\nALL_IPS=$(ALL_IPS)">>onprem-environment)
```

Comment:
Is this needed? How will this be used in non-live ISO environment?

Reply:
Right, this is not needed if we set SERVICE_BASE_URL=http://127.0.0.1:8080. Will remove it. Thank you.